scene/gui/control.cpp

```diff
@@ -73,6 +73,9 @@ Dictionary Control::_edit_get_state() const {
 }
 
 void Control::_edit_set_state(const Dictionary &p_state) {
+	ERR_FAIL_COND((p_state.size() <= 0) ||
```

Comment:
Could you instead use several ERR_FAIL_COND ? One per condition.
This helps debugging the issue when a problem arises.

Reply:
Well in the previous PR I asked them to use a single condition :)

Problems should not arise here, this is an internal method which just happens to be exposed due to a design limitation, but there's no reason for those errors to be triggered unless passing garbage input on purpose to crash then engine ;)

Reply:
Ok, both are fine to me anyway. We can go for the current solution.